modules/dnn/src/layers/blank_layer.cpp

```diff
@@ -111,8 +111,12 @@ class BlankLayerImpl CV_FINAL : public BlankLayer
     virtual Ptr<BackendNode> initInfEngine(const std::vector<Ptr<BackendWrapper> >& inputs) CV_OVERRIDE
     {
         InferenceEngine::DataPtr input = infEngineDataNode(inputs[0]);
-        CV_Assert(!input->dims.empty());
-
+#if INF_ENGINE_VER_MAJOR_LE(2019010000)
+    CV_Assert(!input->dims.empty());
+#else
+    std::vector<size_t> dims = input->getTensorDesc().getDims();
+    CV_Assert(!dims.empty());
+#endif
```

Comment:
Can we use `getTensorDesc` with older versions? Looks like it was available in 2018R5 and 2019R1 as well.